src/theme/shared/size.ts

```diff
@@ -0,0 +1,9 @@
+const size = {
```

Comment:
When checking the `theme` object from `useTheme()`, I can see there's duplication in our setup, regarding `size` and `space`:

![image](https://user-images.githubusercontent.com/3179244/170171175-a20021ac-a95a-4efa-b13a-620905604079.png)

The original attributes from Chakra-UI are `sizes` and `space`, the customs were added by us are `size` and `spacing`. Is that on purpose?

Reply:
I will adjust the names as in Chakra. Thanks